dependencies_test.go

```
@@ -0,0 +1,528 @@
+
+	evidence, err := plugin.EvaluatePolicies(t.Context(), data, deps, []string{policyDir}, nil)
+	if err != nil {
+		t.Fatalf("EvaluatePolicies returned error: %v", err)
+	}
+	if len(evidence) != 2 {
+		t.Fatalf("expected one evidence per dependency, got %d", len(evidence))
+	}
+
+	byDependency := map[string]*proto.Evidence{}
+	for _, ev := range evidence {
+		byDependency[ev.Labels["dependency"]] = ev
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field ev.Labels, use ev.GetLabels() instead

(protogetter)